src/sentry/tagstore/multi/backend.py

```diff
@@ -39,8 +40,18 @@ def worker():
                 (func, args, kwargs) = q.get()
                 try:
                     func(*args, **kwargs)
+                    metrics.incr(
+                        'tagstore.multi.runner',
```

Comment:
I'd append a `.execute` to these two to separate the scheduling from the executing.